### ISSUE TITLE
Student Table in the New Labor Status Form Page Has Too Many columns

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/lsf#67]
**[Original report](https://bitbucket.org/laborstudents/lsf-flask/issue/67) by Alex Bryant (Bitbucket: [Alex Bryant](https://bitbucket.org/Alex Bryant), ).**

----------------------------------------

The “Job Type” column in the table that gets created when the “add student” is clicked on the New Labor Status Form page will still be there when it is a break.  The JavaScript should handle a deletion of that column if the term is a break.

**Comment by bryantal** [comment]
**Original comment by Katelyn Adams (Bitbucket: [paesh1997](https://bitbucket.org/paesh1997), GitHub: [paesh1997](https://github.com/paesh1997)).**

----------------------------------------

@{5cfe56a6765c560c5d88ba1d} status on this issue

**Comment by bryantal** [comment]
**Original comment by Alex Bryant (Bitbucket: [Alex Bryant](https://bitbucket.org/Alex Bryant), ).**

----------------------------------------

While Luis and I have been working on refactoring the JS for the New Labor Status Forms page, we were able deal with this issue.  I’m pretty sure Scott said at one point to us that it was okay to leave it as always secondary in this column when it is a break, so that is what we did.  Whenever we make a pull request for issue #68 then this one would be resolved along with it if it was merged.
